docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md

```diff
@@ -21,10 +21,42 @@ filter by the parent resource. See below for examples of summaries in V3.
 
 - To fetch all apps in a space, use `GET /v3/apps?space_guids=<space-guid>`.
   Passing `include=space` will include the space resource in the response body.
+- To fetch all service offerings in a space use `GET
+  /v3/service_offerings?space_guids=<space-guid>`. 
 - To fetch all service instances in a space use `GET
   /v3/service_instances?space_guids=<space-guid>`. Use the
   experimental `fields` parameter to include related information in the response
   body.
+  
+##### Replacing the space summary response for service instances
+
+Similar fields to what `/v2/spaces/:guid/summary` was offering for services are available in using endpoint in v3.
```

Comment:
"for services are available in using endpoint in v3." should be something like "for services are available using the `service_instances` in `v3`."

Reply:
actually they are available from different endpoints. I will update it to `are available from endpoints in v3`